src/js/engagement_view/src/components/engagementView/sidebar/tables/lensTable/selectLens.tsx

```diff
@@ -11,21 +10,19 @@ import { SelectLensProps } from "types/LensAndNodeTableTypes";
 export function SelectLens(props: SelectLensProps) {
     const classes = useStyles();
 
-    const [btnClass, setBtnClass] = useState(false);
-
     return (
-        <TableRow className={classes.tableRow} key={props.uid}>
+        <TableRow
+            hover
+            key={props.uid}
+            onClick={() => {
+                props.setSelectedLens(props.uid);
+                props.setLens(props.lens);
```

Comment:
more proof that, yeah, setSelectedLens should be named setSelectedLensId or setSelectedLensUid or something - the fact that `setSelectedLens` and `setLens` take two different types isn't great.

Reply:
done